snapshot_test.go

```diff
@@ -2,29 +2,17 @@ package restic_test
 
 import (
 	"testing"
-	"time"
 
 	"github.com/restic/restic"
-	"github.com/restic/restic/server"
 	. "github.com/restic/restic/test"
 )
 
-func testSnapshot(t *testing.T, s *server.Server) {
-	var err error
-	sn, err := restic.NewSnapshot([]string{"/home/foobar"})
-	OK(t, err)
-	// sn.Tree, err = restic.Blob{ID: backend.ParseID("c3ab8ff13720e8ad9047dd39466b3c8974e592c2fa383d4a3960714caef0c4f2")}
-	// ok(t, err)
-	sn.Time, err = time.Parse(time.RFC3339Nano, "2014-08-03T17:49:05.378595539+02:00")
```

Comment:
I don't know what this is... was this even testing anything? I removed it... also removed the commented lines here.. if it's important that this gets done, we should add a github issue